CODE_OF_CONDUCT.md

```diff
@@ -25,7 +25,7 @@ community include:
 * Focusing on what is best not just for us as individuals, but for the
   overall community
 
-Examples of unacceptable behavior include:
+Examples of unacceptable behaviour include:
```

Comment:
This isn't a spelling mistake. It's US English and you're following UK English. Both are correct and can be used in the preferred language style.